style.css

```diff
@@ -106,7 +106,7 @@
     left: 0;
     background: #6070ff;
     width: 100vw;
-    height: 35rem;
+    height: 100vh;
```

Comment:
## [OPTIONAL]
- [ ] Nice work 👏  implementing the changes required by the previous reviewer. But I think your project would look very nice if you made the mobile menu fixed and when the user navigates it disappears. Follow this screencast for more details. 

https://drive.google.com/file/d/1nIOb2Xm_yttZr92aPm7DMmnJl0ENabtW/view?usp=sharing